README.md

```diff
@@ -14,14 +14,19 @@ prerequisite for developing FastBoot apps.
 
 ### Command Line
 
-You can run the FastBoot server from the command line:
+You can run the FastBoot server from the command line by providing the HTML file as well as
+your `vendor.js` and `app.js` that have been built with `ember fastboot:build`. You can also pass
+addition `js` files. Each file will be loaded in the order it appears in the command line.
 
-```
+```sh
 $ ember-fastboot \
-    --port 80
-    --app-file path/to/app-file.js
-    --vendor-file path/to/vendor-file.js
-    --html-file path/to/index.html
+    --host 0.0.0.0 \
+    --port 80 \
+    --html-file path/to/index.html \
+    path/to/vendor-file.js \
+    path/to/app-file.js \
+    path/to/other-file-1.js \
+    path/to/other-file-2.js
```

Comment:
Also how does the `--dist-file` option interact with the list of files? Seems like they are in conflict.